libs/src/oracle2/services/oracles/factory.ts

```diff
@@ -22,25 +17,10 @@ export const Factory =
   (config: Config): ServiceFactory =>
   (handlers: Handlers): Service => {
     const services: Service[] = config.map((config) => {
-      if (config.source === "gql" && config.type === "Optimistic Oracle V1") {
```

Comment:
yeah I completely agree. I was rushing because I was concerned about blocking you and I cut corners that I should not have. I will do better in future